app/waf/src/services/asg.service.ts

```diff
@@ -304,6 +313,21 @@ export class ASGManager {
     );
   }
 
+  async getLicense(trustDeviceId: string): Promise<string> {
```

Comment:
Do you need to add this function?

Reply:
This should be due to the merge issue.

Reply:
If it does not belongs to your change set, please remove it.

Reply:
remove it.